.github/workflows/frontend-docker.yaml のCIが失敗している

継続して失敗しており、再現性あると思われる。対応が必要。 https://github.com/npocccties/chiloportal/actions/workflows/frontend-docker.yaml

日数が経過していてログが取得できない https://github.com/npocccties/chiloportal/actions/runs/6765657710/job/18385603007

以下をみると存在しないDockerイメージを取得しようとして失敗しているようにみえる

```
Run docker build -t frontend .
#0 building with "default" instance using docker driver

#1 [internal] load build definition from Dockerfile
#1 transferring dockerfile: 1.16kB done
#1 DONE 0.0s

#2 [internal] load .dockerignore
#2 transferring context: 77B done
#2 DONE 0.0s

#3 [auth] library/node:pull token for registry-1.docker.io
#3 DONE 0.0s

#4 [internal] load metadata for docker.io/library/node:20.9.0-slim
#4 CANCELED

#5 [internal] load metadata for gcr.io/distroless/nodejs:20
#5 ERROR: gcr.io/distroless/nodejs:20: not found
------
 > [internal] load metadata for gcr.io/distroless/nodejs:20:
------
Dockerfile:27
--------------------
  25 |     RUN corepack yarn workspaces focus --all --production
  26 |     
  27 | >>> FROM gcr.io/distroless/nodejs:20
  28 |     WORKDIR /app
  29 |     ENV NODE_ENV=production
--------------------
ERROR: failed to solve: gcr.io/distroless/nodejs:20: gcr.io/distroless/nodejs:20: not found
Error: Process completed with exit code 1.
```

https://github.com/npocccties/chiloportal/actions/runs/6872341098/job/18690653301#step:6:34

以下から存在するDockerイメージを選択して使用する必要があるようにみえる

> Node.js v20 (gcr.io/distroless/nodejs20-debian12, gcr.io/distroless/nodejs20-debian11) and its dependencies.

https://github.com/GoogleContainerTools/distroless/tree/main/nodejs

--- a/frontend/Dockerfile
+++ b/frontend/Dockerfile
@@ -18,7 +18,7 @@ COPY schemas/ ./schemas
 COPY styles/ ./styles
 COPY templates/ ./templates
 ADD https://raw.githubusercontent.com/npocccties/chiloportal/develop/backend/doc/openapi.yaml /backend/doc/openapi.yaml
-RUN echo "ae3b23e07629a49cd07acdc46e4754c2  ../backend/doc/openapi.yaml" > md5sum.txt && md5sum -c md5sum.txt
+RUN echo "a9cfc6c94f55cdeac753c895f05760f9  ../backend/doc/openapi.yaml" > md5sum.txt && md5sum -c md5sum.txt
 RUN corepack yarn build
 
 FROM base as install

--- a/frontend/Dockerfile
+++ b/frontend/Dockerfile
@@ -24,7 +24,7 @@ RUN corepack yarn build
 FROM base as install
 RUN corepack yarn workspaces focus --all --production
 
-FROM gcr.io/distroless/nodejs:20
+FROM gcr.io/distroless/nodejs20-debian12
 WORKDIR /app
 ENV NODE_ENV=production
 COPY public/ ./public

